spec/integration/user_views_spec.rb

```diff
@@ -88,33 +88,39 @@
   end
 
   describe "the order view" do
-    it "shows the order total" do
+
+    before(:each) do
       user = create(:user)
       allow_any_instance_of(ApplicationController). to receive(:current_user).
       and_return(user)
       create_one_item_with_one_category
       add_item_five_times_to_cart
-      visit cart_path
 
+      visit cart_path
       click_link_or_button("Checkout")
-
+    end
+    it "shows the order total" do
       expect(page).to have_content("Order Summary")
       expect(page).to have_content("Order Total: $#{5 * @item.unit_price / 100}")
     end
 
     it "shows the order time and status " do
-      user = create(:user)
-      allow_any_instance_of(ApplicationController). to receive(:current_user).
-      and_return(user)
-      create_one_item_with_one_category
-      add_item_five_times_to_cart
-      visit cart_path
-
-      click_link_or_button("Checkout")
+      expect(page).to have_content("Order placed at: ")
+      expect(page).to have_content("Current status: ordered")
+    end
 
+    it "shows the order time and status" do
       expect(page).to have_content("Order placed at: ")
       expect(page).to have_content("Current status: ordered")
     end
+
+    it "shows links for each order item" do
+      within('table') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.